bibs/Wuerzburg.json

```diff
@@ -31,6 +31,7 @@
             "status": 4
         },
         "mediatypes": {
+            "0": "ALL",
```

Comment:
mediatypes look wrong (see longer comment above), at least `ALL` needs to be removed.

Reply:
See my [comment](https://github.com/opacapp/opacapp-config-files/pull/11#discussion_r430272819) at Essen.

Reply:
see reply above.